### ISSUE TITLE
Can not reproduce CodeBERT baseline results on test set

### ISSUE THREAD

**Issue by EricLee8** [jadecxliu/CodeQA#3]
Hi, thank you for your remarkable work and the released dataset. I am currently trying to reproduce the CodeBERT baseline results with your recently released code, it works fine on validation set and I am able to get roughly the same results on it. However, when it come to test set, the results are far below the reported values.
Specifically, the best result on validation set are: 
```{dev set: bleu = 33.34 | rouge_l = 31.16 | meteor = 11.17 | EM = 7.50 | Precision = 39.62 | Recall = 30.01 | F1 = 32.06}```
which is close to the reported values.
The corresponding test results are:
```{test set: bleu = 26.19 | rouge_l = 4.71 | meteor = 1.23 | EM = 0.40 | Precision = 6.74 | Recall = 4.69 | F1 = 4.86}```
which is far below the reported values.
Could you please rerun the code or check the test set to verify it is the correct version? Thank you very much.

**Comment by EricLee8** [comment]
It is because of my fp16 training code, now I am able to get the correct results. Thanks :)
